x-pack/plugins/alerting/server/routes/bulk_edit_rules.ts

```diff
@@ -55,6 +57,20 @@ const operationsSchema = schema.arrayOf(
         ])
       ),
     }),
+    schema.object({
+      operation: schema.oneOf([schema.literal('set')]),
+      field: schema.literal('snoozeSchedule'),
```

Comment:
I am wondering if we just call it snooze because we have snooze immediately and schedule working together behind our data model.

Reply:
I took a look at this, the only reason why I think we should keep 'snoozeSchedule' because in `rules_client.tsx -> BulkEditFields` because we are using rule properties as the field for bulk edits, so there is existing precedence for this pattern.